src/Npgsql/MetricsReporter.cs

```diff
@@ -60,7 +64,8 @@ static MetricsReporter()
         CommandDuration = Meter.CreateHistogram<double>(
             "db.client.commands.duration",
             unit: "s",
-            description: "The duration of database commands, in seconds.");
+            description: "The duration of database commands, in seconds.",
+            advice: ShortHistogramAdvice);
```

Comment:
Hmm, it looks like this API was added with .NET 9.

@roji are we going to continue supporting .NET 8 with Npgsql 10? As otherwise we have to add an explicit reference to `System.Diagnostics.DiagnosticSource` package.